### PR TITLE
feat(ui-ux): show progress bar of votes on proposals overview screen

### PR DESCRIPTION
**Pull request by jordankzf**
#### What this PR does / why we need it:

- No votes preview for proposals overview page

#### Which issue(s) does this PR fixes?:

Fixes [WEB-268](https://birthday.atlassian.net/browse/WEB-268)

#### Sample Links & Screenshots:

Link:

<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on multiple web browsers
- [x] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests\*
- [ ] Added e2e tests\*

[WEB-268]: https://birthday.atlassian.net/browse/WEB-268?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/governance/_components/ProposalCard.tsx [hunk 1]
```diff
@@ -23,14 +23,19 @@ import { useCycleEndDate } from "../shared/useCycleEndTime";
...
+import { VoteCount } from "../shared/getVoteCount";
 
 export function ProposalCards({
   proposals,
+  proposalsVotes,
   currentBlockHeight,
   currentBlockMedianTime,
   userQueryProposalStatus,
 }: {
   proposals: GovernanceProposal[];
```
Comment (jordankzf) [inline src/pages/governance/_components/ProposalCard.tsx:37]: A `proposal` is of the `GovernanceProposal` type which is fixed (defined in whale api repo).

For that reason, proposals votes is its own prop in the format of an object, to allow performant subscripting. 'Smuggling' a proposal's votes into `queryProposal` is not possible without overwriting the type declaration via typecasting.

File: src/pages/governance/_components/VotingResult.tsx [hunk 2]
```diff
@@ -16,6 +16,235 @@ import { CircularCheckIcon } from "./CircularCheckIcon";
...
+            comparatorValue={percYes}
+            proposalStatus={undefined}
+            comparingMinimumValue
+            suffix="%"
+            decimalPlace={2}
+          />
+
+          <LabelWithInfoTooltipAndChecks
+            labelTitle="Min. required votes"
+            value={new BigNumber(1)}
```
Comment (kevzzsk) [inline src/pages/governance/_components/VotingResult.tsx:72]: Min. required votes should not be a constant?

File: src/pages/governance/_components/VotingResult.tsx [hunk 3]
```diff
@@ -16,6 +16,235 @@ import { CircularCheckIcon } from "./CircularCheckIcon";
...
+        <div className="p-3 gap-y-4 grid">
+          <LabelWithInfoTooltipAndChecks
+            labelTitle="Min. required approval rate"
+            value={
+              new BigNumber(
+                proposal.proposal.approvalThreshold.replace("%", "")
+              )
+            }
+            comparatorValue={percYes}
+            proposalStatus={undefined}
```
Comment (kevzzsk) [inline src/pages/governance/_components/VotingResult.tsx:64]: is `undefined` here correct?

**Comment by jordankzf** [comment]
@kevzzsk sorry, `git stash` mistake.

**Review by kevzzsk — APPROVED** [review]

File: src/pages/governance/_components/VotingResult.tsx [hunk 4]
```diff
@@ -16,6 +16,203 @@ import { CircularCheckIcon } from "./CircularCheckIcon";
 import { CircularCrossIcon } from "./CircularCrossIcon";
 import { VoteCount } from "../shared/getVoteCount";
 
+// export function VotePopover(proposal) {
```
Comment (kyleleow) [inline src/pages/governance/_components/VotingResult.tsx:19]: can you add a comment above for context why we are commenting this out for now

**Review by kyleleow — APPROVED** [review]

**Pull request merged**